ext/standard/dl.c

```diff
@@ -110,7 +110,7 @@ PHPAPI int php_load_extension(char *filename, int type, int start_now)
 	char *libpath;
 	zend_module_entry *module_entry;
 	zend_module_entry *(*get_module)(void);
-	int error_type, slash_suffix = NULL;
+	int error_type, slash_suffix;
```

Comment:
I think this change will cause line 154 may use of uninitialized value, should be change to slash_suffix=0

Reply:
Done. Then it should be added to master branch also (I've updated above message to target both branches. Thanks.